Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -77,6 +77,47 @@ final class UIViewControllerExtensionsTests: XCTestCase {
 		}
 		XCTAssertEqual(alertController.preferredAction, alertController.actions[preferredButtonIndex])
 	}
+    
+    func testAddChildViewController() {
+        let parentViewController = UIViewController()
+        let childViewController = UIViewController()
+
+        XCTAssert(parentViewController.childViewControllers.isEmpty == true)
+        XCTAssertNil(childViewController.parent)
+        
+        parentViewController.addChildViewController(childViewController, toContainerView: parentViewController.view)
+        
+        XCTAssert(parentViewController.childViewControllers == [childViewController])
+        XCTAssertNotNil(childViewController.parent)
+        XCTAssertEqual(childViewController.parent, parentViewController)
+    }
+    
+    func testRemoveChildViewController() {
+        let parentViewController = UIViewController()
+        let childViewController = UIViewController()
+        
+        parentViewController.addChildViewController(childViewController)
+        parentViewController.view.addSubview(childViewController.view)
+        childViewController.didMove(toParentViewController: parentViewController)
+        
+        XCTAssert(parentViewController.childViewControllers == [childViewController])
```

Comment:
If you use the new `addChildViewController` method, which you already tested in another function, you don't need these checks, technically. If you're going to use them all together you could even combine the 2 test functions into this 1, where you test both at the same time.